skins/wmde19/src/components/layout/Header.vue

```diff
@@ -1,54 +1,48 @@
 <template>
-	<div class="navbar is-fixed-top has-background-bright has-shadow">
-		<div class="container">
-			<div class="navbar-brand">
-				<a class="navbar-item" href="/">
-					<img :src="imagePath + '/images/logo-horizontal-wikimedia.svg'" alt="Wikimedia Deutschland" width="144" height="29">
-				</a>
-				<a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
-					<span aria-hidden="true"></span>
-					<span aria-hidden="true"></span>
-					<span aria-hidden="true"></span>
-				</a>
-			</div>
+    <div class="navbar is-fixed-top has-background-bright has-shadow">
+        <div class="container">
+            <div class="navbar-brand">
+                <a class="navbar-item" href="/">
+                    <img :src="imagePath + '/images/logo-horizontal-wikimedia.svg'" alt="Wikimedia Deutschland"
+                         width="144" height="29">
+                </a>
+                <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false"
+                   v-bind:class="{ 'is-active': showNavbarBurger }" v-on:click="showNavbarBurger = !showNavbarBurger">
```

Comment:
you can list all classes together in the vue directive like
`v-bind:class="[{ 'is-active': showNavbarBurger }, 'navbar-burger']"`
Also `v-on:click` can be `@click`